src/main/resources/mustache-templates/phenotypes.html

```diff
@@ -0,0 +1,99 @@
+{{>layout/header}}
+
+<div class="container vertical-space">
+	<h3>Patient Info</h3>
+	<ul class="list-unstyled">
+		{{#patient}}
+		<li>Name: <span>{{firstName}} {{lastName}}</span></li>
+		<li>Id: <span id="patient_id">{{id}}</span></li> 
+		{{/patient}}
+	</ul>
+	<h3>Phenotypes</h3>
+	<table id="hposummary" class="table table-bordered" style="width: 100%">
+		<thead class="thead-light">
+			<tr>
+				<th></th>
+				<th>HPO Term Name</th>
+				<th>HPO Term Id</th>
+				<th># Observations</th>
+				<th>First Date</th>
+				<th>Last Date</th>
+			</tr>
+		</thead>
+	</table>
+</div>
+
+
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/jquery/jquery.min.js"></script>
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/datatables/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/datatables/js/dataTables.bootstrap4.min.js"></script>
+<script type="text/javascript">
+	function showLabs(d) {
+		// `d` is the original data object for the row
+		let table = "<div class='card float-left'>"
+				+ "<table class='table-bordered sub-table'>"
+				+ "<thead>"
+				+ "<tr class='table-warning'><th>FHIR Id</th><th>LOINC</th><th>Description</th><th>Value</th><th>Date</th></tr>"
+				+ "</thead>" + "<tbody>";
+		for (i = 0; i < d.observations.length; i++) {
+			let observation = d.observations[i];
+			table += "<tr>" + "<td>" + observation.fhirId + "</td>" + "<td>"
+					+ observation.loincId + "</td>" + "<td>"
+					+ observation.description + "</td>" + "<td>"
+					+ observation.value + "</td>" + "<td>" + observation.date
+					+ "</td>" + "</tr>";
+		}
+		table += "</tbody></table></div>";
+		return table;
+	}
+
+	let patientId = $("#patient_id").html();
+
+	$(document).ready(function() {
+		// Set up table sorting
+		let table = $("#hposummary").DataTable({
+			ajax : {
+				"url" : "/summary/" + patientId
+			},
+			info: true,
+			paging : true,
+			searching : true,
+			columns : [ {
+				"className" : "details-control",
+				"orderable" : false,
+				"data" : null,
+				"defaultContent" : ''
+				}, 
+				{ "data" : "hpoTermName" }, 
+				{ "data" : "hpoTermId" },
+				{ "data" : "count"},
+				{ "data" : "first" },
+				{ "data" : "last"}
+			],
+			aaSorting : []
+		// Do not sort any columns initially
+		});
+
+		// Add event listener for opening and closing details
+		$('#hposummary tbody').on('click', 'td.details-control', function() {
+			let tr = $(this).closest('tr');
+			let row = table.row(tr);
+
+			if (row.child.isShown()) {
+				// This row is already open - close it
+				row.child.hide();
+				tr.removeClass('shown');
+			} else {
+				// Open this row
+				row.child(showLabs(row.data())).show();
+				tr.addClass('shown');
+			}
+		});
+
+	});
+</script>
```

Comment:
Rather than having all of this javascript inline, my preference would be to put in in a separate js file. The script include tag would need to go after the dependencies. You could use a model property to indicate whether the javascript should be included.

Reply:
@lawhead I made changes for this in the latest commit. Can you review and make sure I captured what you were suggesting? As for design, I'm planning on meeting with Liz this morning and can pull you into the discussion if interested.

Reply:
I left a couple minor suggestions, but yes, the structure is what I had in mind. It looks a lot cleaner to me. I think any refinements that come out of your discussion with Liz could be handled in another PR. I may sit in on your meeting if you don't mind.